hangman.py

```diff
@@ -0,0 +1,308 @@
+# Problem Set 2, hangman.py
+# Name:
+# Collaborators:
+# Time spent:
+
+# Hangman Game
+# -----------------------------------
+# Helper code
+# You don't need to understand this helper code,
+# but you will have to know how to use the functions
+# (so be sure to read the docstrings!)
```

Comment:
I think these comments won't be helpful in future for anyone, so we can delete all such comments